public/app/features/explore/QueryRowErrors.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { FunctionComponent } from 'react';
+import { ErrorContainer } from './ErrorContainer';
+import { DataQueryError } from '@grafana/data';
+
+export interface Props {
+  queryErrors: DataQueryError[];
+}
+
+export const QueryRowErrors: FunctionComponent<Props> = (props: Props) => {
+  const { queryErrors } = props;
+
+  const errors = queryErrors.map(error => {
+    return {
+      ...error,
+      refId: null,
```

Comment:
Why are we clearing out the refId here?

Reply:
We use `ErrorContainer` component there that primary role was to display errors that aren't related to any query rows - it calls `getFirstQueryErrorWithoutRefId()` function. I'm gonna move error filtering from ErrorContainer to its parent component (`Explore` component) so we don't need to clear out `refId`.